.platform.app.yaml

```diff
@@ -0,0 +1,99 @@
+name: sylius
+
+type: php:8.1
+
+relationships:
+    database: "db:mysql"
+
+disk: 2048
+
+mounts:
+    "/var/cache": "shared:files/cache"
+    "/var/log": "shared:files/log"
```

Comment:
To my knowledge, `shared:files/log` is outdated syntax, that can be seen in years old examples, but it's not a default syntax in docs anymore. https://docs.platform.sh/add-services/network-storage.html#2-add-the-relationship

```suggestion
    "/var/log": 
       source: local
       source_path: log
```

Reply:
Personally I would remove `files` part from `source_path` as each definition is a separate mounts and there is no point prefixing them.